python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -23,114 +23,119 @@
 from ....utils.access_decorators import requires_role_mine_view, requires_role_mine_create, requires_any_of, MINE_VIEW, MINESPACE_PROPONENT
 from ....utils.resources_mixins import UserMixin, ErrorMixin
 from ....constants import MINE_MAP_CACHE, TIMEOUT_12_HOURS
+from app.api.mines.mine_api_models import MINE_LIST_MODEL, MINE
 # FIXME: Model import from outside of its namespace
 # This breaks micro-service architecture and is done
 # for search performance until search can be refactored
 from ....permits.permit.models.permit import Permit
 
 
-class MineResource(Resource, UserMixin, ErrorMixin):
+class MineListResource(Resource, UserMixin):
     parser = reqparse.RequestParser()
     parser.add_argument(
-        'mine_name', type=str, help='Name of the mine.', trim=True, store_missing=False)
+        'mine_name', type=str, help='Name of the mine.', trim=True, required=True, location='json')
     parser.add_argument(
         'mine_note',
         type=str,
         help='Any additional notes to be added to the mine.',
         trim=True,
-        store_missing=False)
-    parser.add_argument(
-        'tenure_number_id',
-        type=int,
-        help='Tenure number for the mine.',
-        trim=True,
-        store_missing=False)
+        store_missing=False,
```

Comment:
I'm starting to waver on the store_missing flag on posts.... without it it will be defaulted to null, which is what the .get() method will do anyways. I feel like i have advocated the other way. but the store_missing is only needed on the put (where the fields supplied actually matter), i think store_missing=False, should be removed from the ListResources.

Reply:
True. I will remove them from the parser for the posts.